contentcuration/contentcuration/utils/publish.py

```diff
@@ -473,7 +548,6 @@ def create_associated_file_objects(kolibrinode, ccnode):
             local_file=kolibrilocalfilemodel,
         )
 
-
```

Comment:
There should be 2 blank lines between any classes, functions, or variables in a module file.